ninjabot.go

```diff
@@ -4,19 +4,18 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"strconv"
-	"sync"
-
+	"github.com/olekukonko/tablewriter"
 	"github.com/rodrigo-brito/ninjabot/exchange"
 	"github.com/rodrigo-brito/ninjabot/model"
 	"github.com/rodrigo-brito/ninjabot/notification"
 	"github.com/rodrigo-brito/ninjabot/order"
 	"github.com/rodrigo-brito/ninjabot/service"
 	"github.com/rodrigo-brito/ninjabot/storage"
 	"github.com/rodrigo-brito/ninjabot/strategy"
-
-	"github.com/olekukonko/tablewriter"
+	"github.com/schollz/progressbar/v3"
 	log "github.com/sirupsen/logrus"
+	"strconv"
+	"sync"
```

Comment:
Please, can you maintain the imports in order?
We use standard libraries followed by, internal packages, followed by external packages.

Reply:
oh, ok my IDE is not auto sort
please check again